### DISCUSSION TITLE
Colored OSD fonts

### DISCUSSION THREAD

**Discussion by Andy0679** [iNavFlight/inav#8822] [Ideas]
My idea is for colored OSD fonts. Either the choice of font colors from a list or the ability to load coloered fonts of our own. There's actually only one color I'm wanting and that is the green like seen on actual fighter jet HUDs. It's not criticle to anything, just something for the fun of it. Thanks for reading my idea.
![hud-in-crj-200](https://user-images.githubusercontent.com/80185340/220801492-881bb058-a4b3-4697-9518-3a7e83f4feee.png)

**Comment by Jetrell** [comment]
If your talking about the analogue fonts. 
The 7456 chip that performs analogue superimposition on our flight controllers. Does not allow for color or grey scale characters. Only Black or White.

**Reply by Andy0679** [reply]
Oh, yes I was talking about analog.  OK  thanks for the response and explanation.  I appreciate it very much.  Have a great day/night.

**Reply by Andy0679** [reply]
How do I close this discussion?
